setup.cfg

```diff
@@ -39,12 +39,16 @@ install_requires =
     numpy==1.13.3; python_version=='3.5' and platform_machine!='aarch64' and platform_system!='AIX'
     numpy==1.13.3; python_version=='3.6' and platform_machine!='aarch64' and platform_system!='AIX' and platform_python_implementation != 'PyPy'
     numpy==1.14.5; python_version=='3.7' and platform_machine!='aarch64' and platform_system!='AIX' and platform_python_implementation != 'PyPy'
-    numpy==1.17.3; python_version=='3.8' and platform_machine!='aarch64' and platform_python_implementation != 'PyPy'
-    numpy==1.19.3; python_version=='3.9' and platform_python_implementation != 'PyPy'
+    numpy==1.17.3; python_version=='3.8' and platform_machine!='arm64'and platform_machine!='aarch64' and platform_python_implementation != 'PyPy'
+    numpy==1.19.3; python_version=='3.9' and platform_machine!='arm64'and platform_python_implementation != 'PyPy'
 
     numpy==1.19.0; python_version=='3.6' and platform_python_implementation=='PyPy'
     numpy==1.20.0; python_version=='3.7' and platform_python_implementation=='PyPy'
 
+    # arm64 supports Python 3.8 and above requires numpy>=1.20.0
+    numpy==1.20.0; python_version=='3.8' and platform_machine=='arm64'
```

Comment:
We should add a note to this: while Python 3.8 on `arm64` works, there is no way to cross-compile from an `x86_64` machine to `arm64` - that's only possible for Python >= 3.9. Hence effectively it's almost impossible to provide 3.8 wheels, because projects do that on CI - and there's no macOS M1 CI.

Reply:
> We should add a note to this: while Python 3.8 on arm64 works, there is no way to cross-compile from an x86_64 machine to arm64 - that's only possible for Python >= 3.9

Why not?

Reply:
You need a machine with macOS 11, as the final CPython 3.8 binary release _does_ have an experimental Universal2 build, it does not work on macOS 10.9+ like the 3.9 releases. I _think_ once macOS 11 is readily available on CI, it might be easier to compile 3.8 universal or cross-compiled `arm64` binaries.

You can also use the Xcode python 3.8, which is universal, and supports 10.15+, but that will only work as long as Xcode doesn't replace it with 3.9 in the future.

Reply:
(yes, "there is no way to cross-compile" is not true, it's a bit hard due to the issues listed above, but not impossible)

Reply:
Works just fine for numpy. See https://github.com/MacPython/numpy-wheels/pull/115

Reply:
I thought the resolution [here](https://github.com/pypa/cibuildwheel/pull/663#discussion_r628731708) was that CPython 3.8 was build without MACOS_DEPLOYMENT_TARGET and as such wouldn't work. Looks like that's a macOS 10.15 runner, though, for MacPython? Ah, you are building universal2 from `python-3.8.10-macosx10.9.pkg`. I didn't realize you could build a universal2 build from a non-universal Python. Is there a way I can download the wheel artifact and try it out on my M1?

Reply:
(I really wish I'd left a machine on 10.15 for tests like this...)

Reply:
> Is there a way I can download the wheel artifact and try it out on my M1?

If that PR is merged, then yes. cc @mattip